inspirehep/modules/multieditor/ext.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Multieditor extension."""
+
+from __future__ import absolute_import, division, print_function
+
+from .views import blueprint
+from .api import blueprint as api_blueprint
+from . import config
+
+
+class MultiEditor(object):
+    def __init__(self, app=None):
+        if app:
+            self.init_app(app)
+
+    def init_app(self, app):
+        app.register_blueprint(blueprint)
+        app.extensions['inspire-multieditor'] = self
+
+
+class ApiMultiEditor(object):
+    def __init__(self, app=None):
+        if app:
+            self.init_app(app)
+
+    def init_app(self, app):
+        self.init_config(app)
+        app.register_blueprint(api_blueprint)
+        app.extensions['inspire-multieditor'] = self
+
+    def init_config(self, app):
```

Comment:
Yes this configuration is only needed in the api side.Actually we dont need any app config because right now the only endpoint in app is the view template

Reply:
If it's not needed, then remove :), the best code is no code 👍